Dockerfile

```diff
@@ -20,33 +20,40 @@ RUN /opt/conda/envs/python2/bin/pip install omego && \
 # scipy-notebook only includes python3 packages
 RUN conda install --name python2 --quiet --yes \
     bokeh \
+    ipywidgets \
     joblib \
     markdown \
     matplotlib \
     pandas \
     pillow \
+    psutil \
     pytables \
     pytest \
     python-igraph \
-    seaborn \
     scikit-image \
     scikit-learn \
-    scipy
+    scipy \
+    seaborn
 
 # RISE: "Live" Reveal.js Jupyter/IPython Slideshow Extension
 # https://github.com/damianavila/RISE
+RUN conda install --quiet --yes -c damianavila82 rise
+
 RUN conda install --name python2 --quiet --yes -c bioconda zeroc-ice && \
-    conda install --name python2 --quiet --yes -c damianavila82 rise && \
     conda install --name python2 --quiet --yes -c pdrops pygraphviz && \
     /opt/conda/envs/python2/bin/pip install \
         graphviz \
         gseapy \
         py2cytoscape \
         pydot \
-        tqdm
+        tqdm \
+        git+https://github.com/IDR/idr-py@master
```

Comment:
Can now be switched to `idr-py`

Reply:
Updated to `idr-py==0.1`, though we also have na idr repo:
https://pypi.python.org/pypi/idr
https://pypi.python.org/pypi/idr-py